tunnel.go

```diff
@@ -137,6 +130,46 @@ func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, con
 	return t, nil
 }
 
+// NewTunnel creates a Tunnel client with given options.
+func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, config *Config) (*Tunnel, error) {
+	m, err := nkn.NewMultiClient(account, identifier, config.NumSubClients, config.OriginalClient, config.ClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	<-m.OnConnect.C
+
+	return newTunnel(account, from, to, tuna, config, m)
+}
+
+type FromTo struct {
```

Comment:
I don't think we need this struct. Using `[]string` and validate len should be enough

Reply:
Yes, removed